reagurk/public/index.html

```diff
@@ -4,6 +4,15 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+<!-- new meta tags-->
+      <meta property="og:type" content="website">
+      <meta property="og:url" content="<Your website link>">
+      <meta property="og:title" content="Gurkult">
+      <meta property="og:description" content="The Gurkan open source community website.">
+      <meta property="og:image" content="https://i.ibb.co/S0KVsg9/logo.png">
```

Comment:
```suggestion
      <meta property="og:image" content="https://gurkult.com/logo.png">
```

Reply:
### I have set the meta tags and made it second inline styling
```
<meta property="og:type" content="website">
      <meta property="og:url" content="https://gurkult.com">
      <meta property="og:title" content="Gurkult">
      <meta property="og:site_name" content="Gurkult" />
      <meta property="og:locale" content="en_US" />
      <meta property="og:description" content="The Gurkult is an open source community website.">
      <meta property="og:image" content="https://gurkult.com/logo.png">
      <meta propert="og:image:alt" content ="Gurkult Logo">
      <meta name="twitter:card" content="summary_small_image"/>
      <meta property="twitter:image" content="https://gurkult.com/logo.png">
      <meta property="twitter:title" content="Gurkult">
      <meta property="twitter:image:alt" content="Gurkult Logo">
      <meta name="theme-color" content="#10B981" />
```
### this was finally obtained in my discord  please review and suggest changes
![](https://i.ibb.co/6FxXT05/Screenshot-599.png)

Reply:
> 
> 
> I believe this should be represented by file path, I do not recall what the exact file path should be in this case. I could be wrong though.

I forgot about this, thank you. It should be `%PUBLIC_URL%/logo.png`